tests/test_abc.py

```diff
@@ -804,3 +805,46 @@ async def test_unexpected_status_code(self):
         with pytest.raises(GraphQLException) as exc:
             await gh.graphql("does not matter")
         assert exc.value.response == response_data
+
+    @pytest.mark.asyncio
+    async def test_response_content_type_parsing_gh121(self):
+        gh, response_data = self.gh_and_response("success-200.json")
+        # Test that a JSON content type still works if formatted without spaces.
+        gh.response_headers["content-type"] = "application/json;charset=utf-8"
+        # Should not fail.
+        resp = await gh.graphql("does not matter")
+        assert resp is not None
+        gh.response_headers["content-type"] = "application/json; charset=utf-8"
+        # Should not fail.
+        resp = await gh.graphql("does not matter")
+        assert resp is not None
+
+    @pytest.mark.asyncio
+    async def test_unknown_response_content_type_gh121(self):
+        gh, response_data = self.gh_and_response("success-200.json")
+        # A non-JSON response should raise an exception.
+        gh.response_headers["content-type"] = "application/gidget;charset=utf-8"
+        with pytest.raises(GraphQLResponseTypeError) as exc:
+            await gh.graphql("does not matter")
+        assert exc is not None
```

Comment:
The exception was caught by the context manager, so no need to check that it isn't `None`.
```suggestion
        with pytest.raises(GraphQLResponseTypeError):
            await gh.graphql("does not matter")
```